miniapps/tools/lor-transfer.cpp

```diff
@@ -125,13 +121,13 @@ int main(int argc, char *argv[])
          lorder = 1;
          cerr << "Switching the H1 LOR space order from 0 to 1\n";
       }
-      fec = new H1_FECollection(order-1, dim);
-      fec_lor = new L2_FECollection(lorder, dim);
+      fec = new H1_FECollection(order, dim);
+      fec_lor = new H1_FECollection(lorder, dim);
    }
    else
    {
       space = "L2";
```

Comment:
Running `./lor-transfer --use-l2` fails with error message `AddMultVWt(...): dimension mismatch`.

Reply:
Thanks for catching this!  I forgot to check l2 spaces after I implemented `ElemMixedMass`.  `ElemMixedMass` was being called with `M_mixed` instead of `M_mixed_el`.  Fixed in the latest commit.